plugins/LilyPad/Tooltips.cpp

```diff
@@ -57,10 +57,23 @@ LPWSTR dialog_message(int ID, bool *updateText)
                    L"This removes the need for manually enabling analog mode with a press of the analog button for games that support, but do not automatically enable analog mode.\n\n"
                    L"Note 1: Analog mode enables the analog sticks to function on a DualShock controller, while in digital mode it behaves as an original PlayStation controller.\n\n"
                    L"Note 2: If analog mode is enabled in a game that does not support it, a game might not recognise a controller is connected.";
-        case ID_TEST:
-            return L"Displays a continuously updated list of the state of all objects on the selected device.\n\n"
-                   L"Use this option to check if all the inputs on a controller function properly.";
+        case IDC_DIAG_LIST:
+            return L"Shows a list of currently available input devices.\n\n"
+                   L"Double-click a device in the list or right-click it and select \"Test Device\" to display a continuously updated list of the state of all inputs on the selected device.\n"
+                   L"Use this option to check if all the inputs on a controller function properly.\n\n"
+                   L"Right-click and select \"Refresh\" to update the list of devices in case a recently connected device has not shown up yet.";
         // Pad tabs
+        case IDC_BINDINGS_LIST:
+            return L"Shows a list ofcurrently bound inputs of the selected Pad.\n\n"
+                   L"Left-click on one the bindings in the list to configure it.\n\n"
```

Comment:
I fixed that already: https://github.com/PCSX2/pcsx2/pull/1679/commits/4b29064cd7af228ff5396a1a6b27f49bb0a42d7d#diff-412bcec8b699ca8df71fcf9e117c9d18R68

Though the VS2013 build seems to have failed without stating what went wrong.